src/renderer/components/dock/pod-logs.tsx

```diff
@@ -78,39 +75,81 @@ export class PodLogs extends React.Component<Props> {
     await this.load();
   }
 
+  /**
+   * Function loads more logs (usually after user scrolls to top) and sets proper
+   * scrolling position
+   * @param scrollHeight previous scrollHeight position before adding new lines
+   */
+  preload = async (scrollHeight: number) => {
+    if (podLogsStore.lines < logRange) return;
+    this.preloading = true;
+    await podLogsStore.load(this.tabId).then(() => this.preloading = false);
```

Comment:
Why is the changing back of `this.preloading` done in a promise chain? If `podLogsStore.load` `throws` then a line after it (without the promise chain) would still fire.

Reply:
No special reason for this, just personally don't like such a structure from aesthetic point of view when you change some flags close to each other:
```
this.ready = false;
await podLogsStore.load(this.tabId);
this.ready = true;
```